src/components/icon/icon.style.ts

```diff
@@ -164,16 +167,18 @@ const StyledIcon = styled.span<StyledIconProps & StyledIconInternalProps>`
           font-size: ${iconConfig.iconSize[fontSize]};
           line-height: ${iconConfig.iconSize[fontSize]};
         `}
-
-        ${type === "services" &&
-        browserTypeCheck(window) &&
+        // FIXME: this can cause hydration mismatches during SSR.
```

Comment:
comment: we should probably add a JIRA ticket or Github Issue to this so that it doesn't get missed